osm2streets-js/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+use serde::{Deserialize, Serialize};
+use wasm_bindgen::prelude::*;
+
+#[derive(Serialize, Deserialize)]
+pub struct Input {
+    driving_side: street_network::DrivingSide,
+}
+
+// TODO Just take a string and bool, maybe remove serde dependency?
```

Comment:
This is nowhere near a final API ready for NPM-land, just enough to get the prototype going

Reply:
Fixed the casing, and good point -- the options struct will come in handy.